packages/elements/src/item/index.ts

```diff
@@ -126,10 +127,20 @@ export class Item extends ControlElement {
   public for: string | null = null;
 
   /**
-   * Cache label element
+   * Reference to the label element
+   */
+
+  private labelRef: Ref<HTMLDivElement> = createRef();
+
```

Comment:
```suggestion
```

Reply:
I'm not sure I understand this suggest, Do you mean I should remove labelRef?

Reply:
There's an empty line under the jsdoc comment

Reply:
Thanks, updated.